Sources/SwiftParser/Statements.swift

```diff
@@ -806,9 +842,14 @@ extension Parser.Lookahead {
   /// Returns `true` if the current token represents the start of a statement
   /// item.
   ///
+  /// - Parameters:
+  ///   - allowRecovery: Whether to attempt to perform recovery.
+  ///   - preferExpr: If `true`, if either an expression or statement could be
+  ///     parsed, returns `false` such that an expression can be parsed.
```

Comment:
This sounds a bit over-complicated to me. What do you think about the following. Same for all the other places that have this doc comment.

```suggestion
  ///   - preferExpr: If either an expression or statement could be
  ///     parsed and this parameter is `true`, the function returns `false` such that an expression can be parsed.
```